libs/styles/src/lib/scss/components/_status-badge.scss

```diff
@@ -13,4 +13,17 @@
   font-size: get-font-size(80);
   line-height: get-line-height(100);
   white-space: nowrap;
+
+  svg {
```

Comment:
Is Stylelint installed and working on your machine? To me (VSCode) this reports a linting error – Avoid tag (type) selectors. Indeed, only classes should be used whenever possible in SCSS code. 

A solution would be to replace `svg` selector with `.badge-icon` class selector (it is already set in `status-badge.component.html` but is not working because no `name` prop is provided to `cvi-ng-icon`).